tools/ci_download_softnpu_machinery

```diff
@@ -19,14 +19,14 @@ fi
 # This is an ASIC administration program.
 if [[ ! -f out/softnpu/scadm ]]; then
     echo "fetching scadm"
-    curl -OL https://buildomat.eng.oxide.computer/wg/0/artefact/01GTD3Y38K0Q14F2989R20QCBA/Ny1N0clhEwz2nfbsdTY8ta7pd9IYy1ofxG6ViCDN6Uy4xW3F/01GTD3YZ0TC9SCB4TFNYWP0DEM/01GTD4CCW2F8Q0NQGD7WEY91D0/scadm
+    curl -OL https://buildomat.eng.oxide.computer/wg/0/artefact/01GYB130ZXZKREJ0SSDXQXTP0F/DZ4rAowUPD7Ir3ZOdHwTDnsk7dtgSgQdRHoiaXJoHdqF29Xo/01GYB13H3Q4P7VQQV83B8TRAPT/01GYB1GET1WH9Y3566636EB27M/scadm
     chmod +x scadm
     mv scadm out/softnpu/
 fi
 
 # Fetch the pre-compiled sidecar_lite p4 program
 if [[ ! -f out/softnpu/libsidecar_lite.so ]]; then
     echo "fetching libsidecar_lite.so"
-    curl -OL https://buildomat.eng.oxide.computer/wg/0/artefact/01GTD3Y38K0Q14F2989R20QCBA/Ny1N0clhEwz2nfbsdTY8ta7pd9IYy1ofxG6ViCDN6Uy4xW3F/01GTD3YZ0TC9SCB4TFNYWP0DEM/01GTD4CA8KRGR8HK4S6B0DEEJC/libsidecar_lite.so
+    curl -OL https://buildomat.eng.oxide.computer/wg/0/artefact/01GYB130ZXZKREJ0SSDXQXTP0F/DZ4rAowUPD7Ir3ZOdHwTDnsk7dtgSgQdRHoiaXJoHdqF29Xo/01GYB13H3Q4P7VQQV83B8TRAPT/01GYB1GDMNCWD3K5MYXP203JYD/libsidecar_lite.so
```

Comment:
I thought ry had added publishing to the `https://buildomat.eng.oxide.computer/public/file/oxidecomputer/...` style links that included the commit?

Reply:
Yup, published artifacts [should now be available](https://github.com/oxidecomputer/sidecar-lite/blob/main/.github/buildomat/jobs/build-and-test.sh#L12-L30). We'll want the ones from the `release` series.

Reply:
I didn't see that, thanks! Will update now.

Reply:
Updated in d37f38469